include/nds/arm9/keyboard.h

```diff
@@ -162,7 +162,7 @@ void keyboardHide(void);
 	\param y the pixel y location
 	\return the key pressed or NOKEY if user pressed outside the keypad
 */
-int keyboardGetKey(int x, int y);
+s16 keyboardGetKey(int x, int y);
```

Comment:
Changing function definitions is probably unnecessary, but it is more consistent. What do you think?

Reply:
I think changing them is more honest, so that people don't use 4 bytes when they can get away with using 2.